modules/comment/comment.service.js

```diff
@@ -19,6 +20,29 @@ class CommentsService {
     return Comment.find({ product: id });
   }
 
+  async getAllCommentsByUser(userEmail) {
+    return Comment.find({ 'user.email': userEmail });
+  }
+
+  async getAllRecentComments({ skip, limit }) {
+    const dateFrom = new Date().getTime();
+    const dateTo = dateFrom - 2592000000;
```

Comment:
Don't use 'Magic numbers'.  Move '2592000000' to separate variable with meaningful name and refer to it here.